elements/pfe-tabs/src/pfe-tab.scss

```diff
@@ -38,17 +41,17 @@ $LOCAL-VARIABLES: (
 
   // Inner style
   background-color: #{pfe-local(BackgroundColor)};
-  color: #{pfe-local(Color)};
+  color: #{pfe-var(ui-disabled--text)};  // @todo replace with pfe(text-muted) in Kendall's color story
 
   cursor: pointer;
-  text-align: #{pfe-local(TextAlign, center, $region: tab)};
-  
+  text-align: #{pfe-local(TextAlign, left, $region: tab)};
+
   // Exposing this for customization
   text-transform: #{pfe-local($cssvar: TextTransform, $region: tab, $fallback: none)};
   font-family:  #{pfe-var(font-family)};
   font-weight:  #{pfe-var(font-weight--normal)};
   font-size:    #{pfe-local(FontSize, $region: tab)};
-    
+
   @for $i from 1 through 6 {
```

Comment:
its not part of this PR, but can we fix / remove this:
https://www.evernote.com/l/AA4w1aXMJwpGnZ_Rqf521LpFd9uWVsTkpDg

we can just remove the wrapper `  @for $i from 1 through 6 {	` and its closing bracket